components/giftbox.vue

```diff
@@ -12,7 +12,7 @@
     <pub-modal-content
       :visible="isModalVisible"
       :title="$t('giftbox.new_features')"
-      @hidden="handleHiddenModal"
+      @hide="handleHiddenModal"
```

Comment:
Nit: Should be `handleHideModal` according to the convention the, no? 😉 Okay for me though, it's clear enough 👍

Reply:
true!